python-for-beginners/11 - Collections/dictionaries.py

```diff
@@ -1,4 +1,7 @@
 person = {'first': 'Christopher'}
+
 person['last'] = 'Harrison'
+# person = {'last': 'Harrison', 'first': 'Christopher'}
```

Comment:
I'm not sure if 'first' comes first or 'last' comes first

Reply:
I thinks it just depend upon you.
Python dictionary is unordered, so there's no rule.

Reply:
Oh cool. I was confused about that.
I will not change the order but add comment about python dictionary
Thanks